.github/workflows/lint_audit.yml

```
@@ -42,8 +42,20 @@ jobs:
         id: changed-files
         run: |
           if [ "${{ github.event_name }}" == "pull_request" ]; then
+            echo "Pull Request detected"
+            echo "PR Head SHA: ${{ github.event.pull_request.head.sha }}"
+            echo "github.sha: ${{ github.sha }}"
+            
```

yamllint

[error] trailing spaces

(trailing-spaces)